### ISSUE TITLE
Honey compatibility issue

### ISSUE THREAD

**Issue by iamsoNewBee** [Roadhog360/Et-Futurum-Requiem#557]
### Initial Questions

- [X] I have searched this issue tracker and there is nothing similar already, and this is not on the [list of known issues](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/146). Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I have verified that this issue occurs in a SUPPORTED environment, meaning I can reproduce this WITHOUT OptiFine, and without Bukkit+Forge server software like KCauldron, Thermos, Crucible, Mohist etc

### Mod Version

etfuturum-2.6.2

### Describe the Issue

As the title,  I wonder if the honey could compat with otther MODs such Forestry or Growcraft? Such get honey from other MOD and kill bees to drop the "MODbees"?

### Conflicting Mods

https://www.curseforge.com/minecraft/mc-mods/forestry
https://www.curseforge.com/minecraft/mc-mods/growthcraft-community-edition

**Comment by Roadhog360** [comment]
Is this a bug report or a question? I can't tell

**Comment by iamsoNewBee** [comment]
BUG report. The honey cant be in common use.

**Comment by iamsoNewBee** [comment]
And I think the bees drop "item" bees could be a way to compat with other mods` beebox? I don`t know. All of these are only suggestions because I dont want to see different kinds of honey on NEI anymore.

**Comment by Roadhog360** [comment]
If you don't really need/want the bee entities you could always just disable them and their respective items. EFR lets you disable mostly anything.

I'll look into better compat. I'm not sure what's going on with your honey bottle recipe there; what are you trying to show me?

**Comment by iamsoNewBee** [comment]
for example, when u take honey form etf  beebox, you get etf honey bottle. And if u take honey from grc beebox, u get grc honey bottle

**Comment by GregoriusT** [comment]
Does that not count as exactly the thing that is supposed to happen?

**Comment by iamsoNewBee** [comment]
It's a growcraft recipe. Just like coppers, many mods has their own coppers but they can be used together. But the honey cant like that because there is no such a ore dictionary. It's really annoying when u see so many different honey generated from different blocks especially in a mod pack. And it could be better if they are all the vanilla* honey. I noticed that the raw ores are unified. If honey could be like that?

**Comment by GregoriusT** [comment]
this one may be dangerous because the bottle is a fluid container.

**Comment by GregoriusT** [comment]
That does not work as well as you would hope. It only works for crafting Recipes like the honey block.

**Comment by iamsoNewBee** [comment]
Yes, it's just an easier way. And for bees, I tried to add a custom drop item for the nest by blockproperties Mod or MTUtils, but all failded.Now I have no idea about the nest.

**Comment by Roadhog360** [comment]
Isn't this a different issue relating to honey bottles? why close this one?

**Comment by iamsoNewBee** [comment]
Oh, sorry. I  mainly focused on the nest recently and forget those bottle things.
